tensorflow/docs_src/community/documentation.md

```diff
@@ -148,30 +148,18 @@ viewing. Do not include url parameters in the source code URL.
 Before building the documentation, you must first set up your environment by
 doing the following:
 
-1. If pip isn't installed on your machine, install it now by issuing the
```

Comment:
You cannot skip this step. You can get rid of codegen, but not the other packages, or you will get incomplete docs.

Reply:
I thought other packages will be downloaded by bazel? I'll add it back.

Reply:
Yeah, not these. These are purely optional dependencies which will enable additional functionality if they are installed on the machine. So if you don't have them installed, TensorFlow will simply not show you some functions. If that happens during doc generation, you get incomplete docs.